doc/faq_debug: qmk cli can also be used to print debug messages (#15528)
qmk/qmk_firmware: docs/faq_debug.md

## Debugging Tools

> There are different tools you can use to debug your keyboard.

Reviewer: ```suggestion
Various tools are available to debug your keyboard.
```

### Debugging With QMK Toolbox

For compatible platforms, [QMK Toolbox](https://github.com/qmk/qmk_toolbox) can be used to display debug messages from your keyboard.

### Debugging with QMK CLI

> The [QMK CLI console command](https://beta.docs.qmk.fm/cli/cli_commands#qmk-console) can be used to display debug messages from your keyboard.

Reviewer: ```suggestion
Prefer a terminal based solution? The [QMK CLI console command](https://beta.docs.qmk.fm/cli/cli_commands#qmk-console) can be used to display debug messages from your keyboard.
```

### Debugging With hid_listen

> Prefer a terminal based solution? [hid_listen](https://www.pjrc.com/teensy/hid_listen.html), provided by PJRC, can also be used to display debug messages. Prebuilt binaries for Windows,Linux,and MacOS are available.

Reviewer: ```suggestion
Something stand-alone? [hid_listen](https://www.pjrc.com/teensy/hid_listen.html), provided by PJRC, can also be used to display debug messages. Prebuilt binaries for Windows,Linux,and MacOS are available.
```

## Debugging Tools

Various tools are available to debug your keyboard.

### Debugging With QMK Toolbox

For compatible platforms, [QMK Toolbox](https://github.com/qmk/qmk_toolbox) can be used to display debug messages from your keyboard.

### Debugging with QMK CLI

> Prefer a terminal based solution? The [QMK CLI console command](https://beta.docs.qmk.fm/cli/cli_commands#qmk-console) can be used to display debug messages from your keyboard.

Reviewer 2: ```suggestion
Prefer a terminal based solution? The [QMK CLI console command](cli_commands.md#qmk-console) can be used to display debug messages from your keyboard.
```
